lib/node_modules/@stdlib/blas/ext/base/dsum/manifest.json

```diff
@@ -35,7 +35,12 @@
 			],
 			"libpath": [],
 			"dependencies": [
-				"@stdlib/blas/ext/base/dsumkbn"
+				"@stdlib/blas/ext/base/dsumkbn",
```

Comment:
@performant23 Would you mind updating this file to use 2-space indentation? Thanks!

Reply:
Hello, @kgryte, thanks for your feedback! I checked the file here and on my local environment (with configuring for whitespace changes with 'git config core.whitespace cr-at-eol' and using 'git diff' and its variants) but both didn't detect any changes. Would you mind checking, please? It's possible that the issue might be related to local environment configurations. I've noticed that all the .json files default to 4 spaces on my setup, as configured that way.

Reply:
You should be using EditorConfig, as mentioned in the dev guide. We have per file type configuration. See the EditorConfig file in the root project directory.

Reply:
Yeah, right! Thanks @kgryte! Have made the changes!

Reply:
Awesome. Thanks! Will review again later today.